CHANGELOG.md

```diff
@@ -41,6 +41,7 @@
 - Check that metricNames are unique in ScaledObject ([#1390](https://github.com/kedacore/keda/pull/1390))
 - Serve OpenAPI spec from KEDA Metrics Apiserver ([#1512](https://github.com/kedacore/keda/pull/1512))
 - Support metrics with multiple dimensions and configurable metricValues on AWS Cloudwatch Scaler ([#1230](https://github.com/kedacore/keda/issues/1230))
+- Add end-to-end tests for OpenStack Swift scaler
```

Comment:
Minor nit, no need to add a separate line, you can link this PR in the existing entry for OpenStack Swift scaler, ie update the line 24 of the Changelog to::
`- Introducing OpenStack Swift scaler ([#1342](https://github.com/kedacore/keda/issues/1342), [#1522](https://github.com/kedacore/keda/pull/1522))`